### PR TITLE
Removed unnecessary apt-get update commands

### PR DESCRIPTION
**Pull request by EliasGabrielsson**
Signed-off-by: Elias Gabrielsson elias@gabson.se

### CODE REVIEW

**Comment by EliasGabrielsson** [comment]
@mstormi could you do a quick review on this?

**Comment by EliasGabrielsson** [comment]
... @BClark09 or @ThomDietrich

File: openhabian-setup.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ for shfile in $BASEDIR/functions/*.bash; do source "$shfile"; done
 if [[ -n "$UNATTENDED" ]]; then
   # apt/dpkg commands will not try interactive dialogs
   export DEBIAN_FRONTEND=noninteractive
-
+  apt-get update
```
Comment (mstormi) [inline openhabian-setup.sh]: This being the first (and only) apt update in unattended mode, it means the following lines are executed without running update before (see last line which is issued when starting openhab-setup from first-boot.bash).
Can you move it into first-boot.bash ?

2019-08-27_18:59:19_JST [openHABian] Flag file 'afterfirstboot.lock' not found, continuing with '/boot/first-boot.bash' script.

2019-08-27_18:59:19_JST [openHABian] Starting the openHABian initial setup.

2019-08-27_18:59:19_JST [openHABian] Storing configuration... OK

2019-08-27_18:59:19_JST [openHABian] Starting webserver with installation log... FAILED

2019-08-27_18:59:24_JST [openHABian] Changing default username and password... SKIPPED

2019-08-27_18:59:24_JST [openHABian] Setting up Ethernet connection... OK

2019-08-27_18:59:24_JST [openHABian] Ensuring network connectivity... OK

2019-08-27_18:59:24_JST [openHABian] Waiting for dpkg/apt to get ready... OK

2019-08-27_18:59:35_JST [openHABian] Updating repositories and upgrading installed packages... OK

2019-08-27_18:59:43_JST [openHABian] Installing git package... OK

2019-08-27_18:59:45_JST [openHABian] Cloning myself... OK

2019-08-27_18:59:45_JST [openHABian] Executing 'openhabian-setup.sh unattended'...

Comment (EliasGabrielsson) [reply openhabian-setup.sh]: `apt-get update` are executed once on line: https://github.com/openhab/openhabian/blob/c0403a4798d6ae4a0194db28bd8ab172be3e1fd9/build-image/first-boot.bash#L132

Comment (EliasGabrielsson) [reply openhabian-setup.sh]: Got you covered. Have it in both places to ensure running openhabian-config.sh directly will work as well. Should be a supported feature. (Don't know if we are really there though.)

**Pull request merged**